foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -27,12 +25,13 @@ def source_type(self):
         return "ARRAY_EXPRESS"
 
     def downloader_task(self):
-        return download_array_express
+        return ("data_refinery_workers.downloaders"
+                + ".array_express.download_array_express")
```

Comment:
Hmm, I'm actually not sure. The way I see it is that this is essentially a constant right now. What do you think the benefits of defining it as a constant instead of as an attribute are? Explicitness about it's status as a constant?